data/transit/route/bus.json

```diff
@@ -10492,7 +10526,11 @@
       "locationSet": {"include": ["fr"]},
       "tags": {
         "network": "TCL",
-        "operator": "TCL",
+        "network:wikidata": "Q8700",
+        "network:wikipedia": "fr:Transports en commun lyonnais",
+        "operator": "Keolis Lyon",
+        "operator:wikidata": "Q61456136",
+        "operator:wikipedia": "fr:Keolis",
```

Comment:
It's better to remove this `operator:wikipedia` if it isn't for the same page that the `operator:wikidata=Q61456136` links to.  Someday if people make a Wikipedia page for "Keolis Lyon" we can add it here instead.

Reply:
same thing.
Keolis Lyon is a subsidiary of Keolis.